_episodes/02-software-development.md

```diff
@@ -19,24 +17,137 @@ View (user interface), and Controller (input/output and data manipulation)."
 - "The software project template we use throughout this workshop is an example of an MVC application that manipulates
 patients’ inflammation data and performs basic statistical analysis using Python."
 ---
-## Introduction
+
+## Our Software Project
+So, you have joined a software development team that has been working on the [patient inflammation project](https://github.com/softwaresaved/python-intermediate-inflammation) developed in Python and stored on GitHub. 
+The software project studies inflammation in patients 
+who have been given a new treatment for arthritis and reuses the inflammation dataset from the [novice Software Carpentry Python lesson](https://swcarpentry.github.io/python-novice-inflammation/index.html). The project is not finished and we will be building 
+on top of the existing code during the workshop. The first thing to do is to obtain a local copy of the project on 
+your machine and inspect it. 
+
+To create your own copy of the software project repository from GitHub:
+
+1. Log into your GitHub account and go to the [template repository URL](https://github.com/softwaresaved/python-intermediate-inflammation).
+![template-repository](../fig/template-repository.png)
+2. Click `Use this template` button towards the top right of the template repository's GitHub page to create a **copy** of
+the repository under your GitHub account. Note that each participant is creating their own copy to work on. Also,
+we are not forking the directory but creating a copy (remember - you can fork only once but can have multiple copies in GitHub).
+3. Make sure to select your personal account and set the name of the project to `python-intermediate-inflammation` (you can call it
+anything you like, but it may be easier if everyone uses the same name). Also set the new repository's visibility to
```

Comment:
Perhaps give a reason for why everyone should use the same name? e.g. "...but it may be easier for future group exercises if everyone uses the same name"

Reply:
Fixed via fec8087.